src/snovault/elasticsearch/index_logger.py

```diff
@@ -0,0 +1,98 @@
+"""
+Indexing Data Log
+
+* Logs uuid progress, timing, and exceptions as they happen.
+"""
+import logging
+import time
+
+
+class IndexLogger(object):
+    '''Basic Logger tailored to our indexing process'''
+    log_name = 'indexing_time'
+    log_path = './src/snovault/elasticsearch'
+
+    def __init__(self, do_log=False, log_name=None, log_path=None):
+        self._do_log = do_log
+        if log_name:
+            self.log_name = log_name
+        if log_path:
+            self.log_path = log_path
+        self._the_log = None
+
+    def _close_handlers(self):
+        '''Close all log handlers'''
+        for handler in self._the_log.handlers:
+            handler.close()
+            self._the_log.removeHandler(handler)
+
+    def _get_log(self):
+        if self._do_log:
+            file_name = "{}-{}.log".format(
+                self.log_name,
+                str(int(time.time() * 10000000)),
```

Comment:
this is converting microsec-> sec?   Probably should have a comment here.

Reply:
Well yes and no, It's just a way to keep timestamps on the file.  I can add a comment anyhow.